seed/tests/test_audit_template.py

```diff
@@ -592,3 +592,104 @@ def test_batch_export_to_audit_template(self, mock_request):
         self.assertEqual(self.view4.id, details[0]["view_id"])
         self.assertEqual(exp, details[0]["message"])
         self.assertEqual("4444", self.state4.audit_template_building_id)
+
+
+# For developer use only. Must be run with live submission data.
+class AuditTemplateSubmissionImport(TestCase):
```

Comment:
These tests were very valuable in the development of the features. They must be run with live data so they aren't relavent for CI, but could be useful to future developers